### PR TITLE
Update infobadge samples to be more accessible

### LINKED ISSUE #769: Bug: XAML Controls Gallery: Accessibility: [MAS4.2.1]: Narrator fails to announce the <count> of Inbox messages when navigated there and Users are unable to navigate to other symbols under 'Different info badge styles' heading.
**Tags**
A11yFeature, Accessibility, A11yMAS, A11ySev2, MAS4.2.1, XAML Controls Gallery (Debug), ScreenReader, Win32, Wipro

**Steps to reproduce the bug**

1. Launch the XAML Controls Gallery (Debug) App
2. Go to search bar and type Info Badge and hit enter key.
3. Now Tab to navigate to the Home tab and then use Down arrow key to navigate to 'Inbox' and listen to Narrator announcement.
4. Narrator fails to announce the <count> of unread/new mails number which is present over inbox button.
5. Now try to navigate to other different info badge symbols using 'Caps+right/Left' arrow keys and observe only one of the info badge symbol is getting focussed but not the other symbol.

**Actual behavior**
Users are unware of the presence of the info badge symbols over the buttons as narrator fails to announce aby information regarding them.
Also few symbols under 'Different info badge symbols' are unable to focus using 'Caps+arrow' keys as well.

**Expected behavior**
Narrator should announce the information about these info badge symbols whenever navigated and users must be able to navigate to all the info badge symbols present in this page.

**Version Info**
Microsoft OS: 22449.1000
App: XAML Controls Gallery (Debug) (Version 1.2.16.0)

**Attachment**

1. inbox count.png
inbox count
2. diff info badge.png
diff info badge

### PR DESCRIPTION
**Pull request by chingucoding**
## Motivation and Context

Fixes #769
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: WinUIGallery/ControlPages/InfoBadgePage.xaml [hunk 1]
```diff
@@ -67,9 +67,11 @@
         <local:ControlExample x:Name="Example2" HeaderText="Different InfoBadge Styles" HorizontalContentAlignment="Stretch" >
             <local:ControlExample.Example>
                 <StackPanel Orientation="Horizontal" contract5Present:Spacing="20" HorizontalAlignment="Center">
-                    <InfoBadge x:Name="infoBadge2" Style="{StaticResource AttentionIconInfoBadgeStyle}" HorizontalAlignment="Right"/>
+                    <InfoBadge x:Name="infoBadge2" Style="{StaticResource AttentionIconInfoBadgeStyle}" HorizontalAlignment="Right"
```
Comment (bpulliam) [inline WinUIGallery/ControlPages/InfoBadgePage.xaml:70]: The xaml here does not match the xaml above (and it didn't before the change :)).

File: WinUIGallery/ControlPages/InfoBadgePage.xaml [hunk 2]
```diff
@@ -86,9 +88,11 @@
 
             <local:ControlExample.Xaml>
                 <x:String xml:space="preserve">
-&lt;InfoBadge x:Name="infoBadge2" Style="{StaticResource $(Style)IconInfoBadgeStyle}" /&gt;
+&lt;InfoBadge x:Name="infoBadge2" Style="{StaticResource $(Style)IconInfoBadgeStyle}"
```
Comment (bpulliam) [inline WinUIGallery/ControlPages/InfoBadgePage.xaml:91]: The xaml here does not match the xaml above (and it didn't before the change :)).

File: WinUIGallery/ControlPages/InfoBadgePage.xaml [hunk 3]
```diff
@@ -86,9 +88,11 @@
...
+    AutomationProperties.Name="Warning badge" AutomationProperties.AutomationControlType="Image"/&gt;
 &lt;InfoBadge x:Name="infoBadge3" Style="{StaticResource $(Style)ValueInfoBadgeStyle}" Value="10" /&gt;
-&lt;InfoBadge x:Name="infoBadge4" Style="{StaticResource $(Style)DotInfoBadgeStyle}" /&gt;
+&lt;InfoBadge x:Name="infoBadge4" Style="{StaticResource $(Style)DotInfoBadgeStyle}"
```
Comment (bpulliam) [inline WinUIGallery/ControlPages/InfoBadgePage.xaml:94]: The xaml here does not match the xaml above (and it didn't before the change :)).

**Comment by chingucoding** [comment]
Sample code should be correct now.

**Comment by Jay-o-Way** [comment]
What's keeping this?

**Review by karkarl — APPROVED** [review]

**Pull request merged**